internal/os/exec/exec.go

```diff
@@ -0,0 +1,8 @@
+package exec
```

Comment:
We should probably bring along (and adapt appropriately) the tests from os/exec too (lp_test.go lp_unix_test.go lp_windows_test.go), I guess.

Reply:
Done. We'll see how the windows tests get on ;-)

Reply:
I had to scrap this in the end because the `os/exec` tests are quite involved and end up pulling in `internal/poll` which is not a rabbit hole we want to explore (for now) I suspect.